packages/ubus-lime-openairview/Makefile

```diff
@@ -14,7 +14,7 @@ define Package/$(PKG_NAME)
   MAINTAINER:=Marcos Gutierrez <gmarcos87@gmail.com>
   SUBMENU:=3. Applications
   TITLE:=Openairview ubus module
-  DEPENDS:= +lua +libubox-lua +libubus-lua +libiwinfo-lua +luci-lib-nixio +luci-lib-json
+  DEPENDS:= +lua +libubox-lua +libubus-lua +libiwinfo-lua +luci-lib-nixio +luci-lib-json +lime-system
```

Comment:
Are you really really sure `lime-system` is needed?

Reply:
https://github.com/libremesh/lime-packages/issues/766#issuecomment-702940526
and
https://github.com/libremesh/lime-packages/blob/5403093ba9fe47dedef1345df5d08cf9f1a34072/packages/ubus-lime-openairview/files/usr/libexec/rpcd/lime-openairview#L14